### ISSUE TITLE
cannot reproduce the accuracy on CUB

### ISSUE THREAD

**Issue by yifanpu001** [raoyongming/CAL#10]
I am trying to reproduce the result of the CUB dataset, which is 90.6 acc ( table 1 in the paper). However, I use the same config and startup script as the code repo, but only get 90.03 acc for the last epoch. I notice that the total training epoch for fgvc task is not reported in the paper. So what is the proper epoch to get the 90.6 acc? Are there any other reasons that could affect reproducing the acc? 

Please see attachment for my training log. Thanks
[train.log](https://github.com/raoyongming/CAL/files/8338086/train.log)
!

**Comment by raoyongming** [comment]
Hi, thanks for your interest in our paper. As mentioned in our paper, the final results reported in Table 1 are based on multiple crops and horizontal flipping. You can use the `infer.py` to obtain the final performance. The log only records the single-view accuracies measured during training.

**Comment by yifanpu001** [comment]
Thank you for your reply! I use the infer.py and the model_bestacc.pth to do the test process. The test acc is 90.37, which is higher than the acc reported during training, but still slightly lower than the result in table 1. How can I further improve the accuracy? Does it cause by the variance between different types of GPU or different random seeds?

**Comment by raoyongming** [comment]
I think the difference may be caused by the environment (GPU types, Pytorch/CUDA versions) or random seeds. In my log, the max accuracy during training is 90.0 and the multi-crop accuracy is 90.6.

**Comment by yifanpu001** [comment]
really appreciate your help!
